src/views/Dashboard/Customer/Home/RequestTest/SelectServicePage/SelectService.vue

```diff
@@ -222,6 +222,13 @@ export default {
 
     toPaymentHistory () {
       this.$router.push({ name: "customer-payment-history" })
+    },
+
+    getImageLink(val){
+      if(val && val != ""){
+        return val
+      }
+      return "https://ipfs.io/ipfs/QmZveuvcQAbkcbcop28iqWrnCMoxN2n5SGzA3Hwr1cJ61i/debio-logo.png"
```

Comment:
The DeBio logo shouldn't be hardcoded like this. Wouldn't it be better to turn it into an Icon? @luthfiazhari.

Please ask @bukandicki for help. @bukandicki, please help @luthfiazhari with this.

Reply:
Do you use this as the default avatar? @luthfiazhari